cremalab/crema-app-web
# Updated test:visual:ci so that it starts and stops storybook automatically

## Description

Updated `test:ci` so that storybook is started on a local server automatically and stopped when needed.

### 📋 Acceptance Criteria

- [x] `test:ci` can be run successfully even when no other script is running

> Checked off by the PR assignee(s)

### 👩‍🔬 Test Instructions

1. Pull down branch
1. Run `npm run test:ci`
1. Ensure it runs successfully 

## 🔎 Review Checklist


> Checked off by the PR reviewers
